src/resizing.jl

```diff
@@ -182,16 +191,43 @@ odims(original, i, short_size) = oftype(first(short_size), axes(original, i))
 
 """
     imresize(img, sz) -> imgr
-    imresize(img, inds) -> imgr
+    imresize(img, ratio::Real) -> imgr
+    imresize(img, inds::Integer...) -> imgr
```

Comment:
given how the names are used in the description we should at least switch `inds` with `sz` here. (because `inds` means that the elements are ranges, while `sz` implies the elements are `Integer`). Technically both arguments can be a tuple or a var-arg, but i get the idea here to contrast the `Integer` to the `Real`